.azure/pipelines/ci-fuzz.yml

```diff
@@ -31,3 +31,17 @@ stages:
       - template: jobs/fuzzing-libFuzzer.yml
         parameters:
           method: Base64_Url_Decode
+
+  - stage: Base64_Whitespace
+    dependsOn: [] # break sequential dependency
+    jobs:
+      - template: jobs/fuzzing-libFuzzer.yml
+        parameters:
+          method: Base64_Default_Decode_Whitespace
+
+  - stage: Base64Url_Whitespace
+    dependsOn: [] # break sequential dependency
+    jobs:
+      - template: jobs/fuzzing-libFuzzer.yml
+        parameters:
+          method: Base64_Url_Decode_Whitespace
```

Comment:
For these jobs the combination of the vectorized pathes don't need to run.
Otherwise it's too many jobs ($ > 10 $).